src/main/java/org/folio/rmapi/RMAPIService.java

```diff
@@ -446,4 +445,27 @@ private String constructURL(String path) {
     LOG.info("constructurl - path=" + fullPath);
     return fullPath;
   }
+
+  public static class VendorResult {
```

Comment:
I guess this class was merged accidentally, on master I moved VendorResult to it's own file, so this can be deleted.

Reply:
Thanks for all your feedback @andrii-paias ..addressed 👍